taxi/views.py

```diff
@@ -17,3 +18,34 @@ def index(request):
     }
 
     return render(request, "taxi/index.html", context=context)
+
+
+class ManufacturerListView(ListView):
+    model = Manufacturer
+    context_object_name = "manufacturer_list"
+    paginate_by = 5
+    template_name = "taxi/manufacturer_list.html"
+    queryset = Manufacturer.objects.all().order_by("name")
+
+
+class CarListView(ListView):
+    model = Car
+    context_object_name = "car_list"
+    paginate_by = 5
+    template_name = "taxi/car_list.html"
+    queryset = Car.objects.all().select_related("manufacturer")
+    queryset = Car.objects.all().prefetch_related("drivers")
```

Comment:
As a result you will have only last queryset. Do:
```suggestion
    queryset = Car.objects.all().select_related("manufacturer").prefetch_related("drivers")
```